src/token.cc

```diff
@@ -420,14 +422,15 @@ void expr_t::token_t::next(std::istream& in, const parse_flags_t& pflags)
         if (in.fail() || ! in.good())
           throw_(parse_error, _("Failed to reset input stream"));
 
-        c = static_cast<char>(in.peek());
-        if (c != -1) {
-          if (! std::isalpha(c) && c != '_')
-            expected('\0', c);
+        int peeked = in.peek();
+        if (peeked == -1) {
+          throw_(parse_error, _("Unexpected EOF"));
+        } else {
+          if (! std::isalpha(peeked) && peeked != '_')
+            expected('\0', peeked);
+	  c = static_cast<char>(peeked);
```

Comment:
You got a tab here which messes up the space indentation used everywhere else, it seems.

Reply:
Oops; I'll fix that.